missing find_dependency calls in AvogadroLibsConfig.cmake.in

With avogadrolibs 1.100 package on openSUSE (built with Qt 6)

https://github.com/OpenChemistry/avogadrolibs/blob/master/avogadro/qtgui/CMakeLists.txt#L127 tells CMake Qt6Widgets and Qt6Concurrent are PUBLIC link targets.

The generated AvogadroLibsTargets.cmake consequently contains:

```
set_target_properties(Avogadro::QtGui PROPERTIES
  INTERFACE_INCLUDE_DIRECTORIES "${_IMPORT_PREFIX}/include;${_IMPORT_PREFIX}/include"
  INTERFACE_LINK_LIBRARIES "Avogadro::IO;Qt6::Widgets;Qt6::Concurrent"
)
```
This also applies to Avogadro::QtOpenGL, which links publicly to Qt6::Widgets;Qt6::OpenGLWidgets and
Avogadro::MoleQueue which links to QtGui;Qt6::Widgets;Qt6::Network

AvogadroLibsConfig.cmake.in only has a find_dependency call for Qt6OpenGLWidgets. Other public link targets must be searched as well or we're seeing errors such as:


```
[   33s] CMake Error at /usr/lib64/cmake/avogadrolibs/AvogadroLibsTargets.cmake:208 (set_target_properties):
[   33s]   The link interface of target "Avogadro::QtGui" contains:
[   33s] 
[   33s]     Qt6::Concurrent
[   33s] 
[   33s]   but the target was not found.  Possible reasons include:
[   33s] 
[   33s]     * There is a typo in the target name.
[   33s]     * A find_package call is missing for an IMPORTED target.
[   33s]     * An ALIAS target is missing.
[   33s] 
[   33s] Call Stack (most recent call first):
[   33s]   /usr/lib64/cmake/avogadrolibs/AvogadroLibsConfig.cmake:29 (include)
[   33s]   CMakeLists.txt:46 (find_package)
```

(this happens when building kalzium)

--- a/cmake/AvogadroLibsConfig.cmake.in
+++ b/cmake/AvogadroLibsConfig.cmake.in
@@ -23,6 +23,11 @@ include(CMakeFindDependencyMacro)
 
 if (@QT_VERSION@ EQUAL 6)
   find_dependency(Qt6OpenGLWidgets)
+  find_dependency(Qt6Widgets)
+  find_dependency(Qt6Core)
+  find_dependency(Qt6Gui)
+  find_dependency(Qt6Network)
+  find_dependency(Qt6Concurrent)
 endif()
 
 if(NOT TARGET AvogadroCore)

